others/recursive_tree_traversal.cpp

```diff
@@ -76,33 +77,34 @@ namespace recursive_tree_traversals {
  * @param right follow up right subtree.
  */
 struct Node {
-    uint64_t data = 0;     ///< The value/key of the node.
-    struct Node *left{};   ///< struct pointer to left subtree.
-    struct Node *right{};  ///< struct pointer to right subtree.
+    std::uint64_t data = 0;  ///< The value/key of the node.
+    struct Node *left{};     ///< struct pointer to left subtree.
+    struct Node *right{};    ///< struct pointer to right subtree.
 };
 /**
  * @brief BT used to make the entire structure of the binary tree and the
  * functions associated with the binary tree
  */
 class BT {
  public:
-    std::vector<uint64_t>
+    std::vector<std::uint64_t>
```

Comment:
Is prefixing uint64_t necessary? Most of the algorithms don't do it. Thinking about it might become a future problem if we don't prefix it. Would love to hear your opinion on doing
```cpp
using std::uint64_t;
```

Reply:
I would either go for `uint64_t` or `std::uint64_t`. I prefer the second one, since it ensures that the _standard_ type is used.

Please let me know if you insist on changing it here.